.github/ISSUE_TEMPLATE/public-websites-intake.md

```diff
@@ -10,57 +10,64 @@ assignees: karak888
 # WIP
 
 ### Instructions
-- All requests for work needing to be fulfilled by a Public Websites resource must submit this intake request form. The team's capacity and the type of request will determine which sprint the work falls into, so it is highly recommended that requests are submitted as soon as you're aware you will need help. Different types of requests may require more time in advance.
+- All teams requesting support or work that needs to be fulfilled by a Public Website resource must submit this intake request form. The team's capacity and the type of request will determine which sprint the work falls into.
```

Comment:
Consider leading off with a list of the types of situations/requests a team would have of Public Websites.